datalad/support/annexrepo.py

```diff
@@ -335,3 +336,103 @@ def annex_add_to_git(self, files):
             self.cmd_call_wrapper.run(cmd_list, cwd=self.path)
         else:
             self.git_add(files)
+
+    def annex_initremote(self, name, options):
+        """Creates a new special remote
+
+        Parameters:
+        -----------
+        name: str
+            name of the special remote
+        """
+        # TODO: figure out consistent way for passing options + document
+
+        cmd_list = ['git', 'annex', 'initremote', name] + options
+        self.cmd_call_wrapper(cmd_list, cwd=self.path)
+
+    def annex_enableremote(self, name):
+        """Enables use of an existing special remote
+
+        Parameters:
+        -----------
+        name: str
+            name, the special remote was created with
+        """
+
+        cmd_list = ['git', 'annex', 'enableremote', name]
+        self.cmd_call_wrapper(cmd_list, cwd=self.path)
+
+    @normalize_paths
+    def annex_addurl_to_file(self, file, url, options=[]):
+        """Add file from url to the annex.
+
+        Downloads `file` from `url` and add it to the annex.
+        If annex knows `file` already, records that it can be downloaded from `url`.
+
+        Parameters:
+        -----------
+        file: str
```

Comment:
imho mode  is too  important ( fast,  relaxed)  to just proxy it within  options...   also pathdepth... co uld be wrong

 but in those we don't even download anything.   if file is already present,  nothing is downloaded